objectives-and-ambition.tex

```diff
@@ -36,14 +36,21 @@ \subsubsection{State of the Art}
 mistake is made in trying to follow the protocol. Second, the automatic
 execution saves time.
 
-\emph{Given enough training or experience}, researchers can use existing technical
-solutions to compose such archives, and to allow a completely automatic
-reproduction of the necessary software environment, and then reproduction of the actual
-result using that software.
+There are two approaches emerging to achieve this reproducibility: (a) to use a
+workflow tool or environment that caters for the given use case (for example
+~\cite{reana2019} \TODO{cite other workflow systems}). Or (b) the use standard
+(software engineering) computing tools and conventions (git, make, python, perl,
+bash, \ldots) to specify the compute environment and automate reproduction
+computation.
+
+The workflow tool approach (a) has limitations in that the tool will not support
+all use cases, and there may be a vendor lock-in. The standard computing tools
+approach (b) is generic but not accessible to all scientists as it requires
+sufficient training or experience to be effective.
```

Comment:
```suggestion
The workflow tool approach (a) is more robust, but requires `all-in' adoption by authors and reproducers alike,
and may not suit all use cases. It can also have associated `vendor lock-in' effects
- once a tool is adopted for one piece, it must be used for all associated work.
The standard computing tools approach (b) is generic, but not accessible to all researchers
 as it requires sufficient training or experience to be effective.
 The loose coupling of tools and modular choices make it more flexible (covering more use cases),
 but more difficult to follow robustly.
```